data/projects/documentation_extraction.md

```diff
@@ -0,0 +1,44 @@
+Code needs documentation. Most programming languages have some form of inline
+documentation standard.
+
+Writing documentation is hard. After writing the documentation, things like
+alignment and grammar are harder to look through. While the programmer writes
+the documentation, coala could help the programmer conform to the documentation
+standard and other trivial issues such as capitalization and grammar.
+
```

Comment:
This is the most important part from what it sounded like from stephanie at fosdem.
My take on the story thingy:

"Code needs documentation. Everyone knows that, but for some reason it doesn't end up having one.
This probably is partly due to the fact that having a second place to write documentation makes it easy to "forget" about it. To solve this problem, and also to offer developers easier access to some important information, many languages support having documentation in source files. The problem with this is, that most linters only work on code or pure documentation files but can not lint documentation in source files. This leads to a lack of checks for in source documentation which then can lead to bad documentation due to formatting issues and the like.